src/main/java/hudson/plugins/ws_cleanup/WsCleanupMatrixAggregator.java

```diff
@@ -47,6 +48,8 @@ public boolean endBuild() throws InterruptedException, IOException {
     }
 
     private boolean shouldCleanBuildBasedOnState(Result result) {
```

Comment:
Good practice: Use CheckForNull to document the possible Null parameter for method users